html/includes/api_functions.inc.php

```diff
@@ -1038,10 +1040,18 @@ function list_alerts()
     $sql = '';
     if (isset($router['id']) && $router['id'] > 0) {
         $alert_id = mres($router['id']);
-        $sql      = 'AND `A`.id=?';
+        $sql      = ' AND `A`.id=?';
         array_push($param, $alert_id);
     }
 
+    if (isset($_GET['severity'])) {
```

Comment:
Change the `$_GET` use to `$vars` then you don't need mres + we sanitize the input.